apps/gp2-server/test/data-providers/user.data-provider.test.ts

```diff
@@ -122,16 +170,68 @@ describe('User data provider', () => {
       ).not.toBeDefined();
       expect(nock.isDone()).toBe(true);
     });
-    test('Should update the region and last name through a clean-update', async () => {
+    test.each`
+      region                      | expected
+      ${'Africa'}                 | ${UsersDataRegionEnum.Africa}
+      ${'Asia'}                   | ${UsersDataRegionEnum.Asia}
+      ${'Australia/Australiasia'} | ${UsersDataRegionEnum.AustraliaAustraliasia}
+      ${'Europe'}                 | ${UsersDataRegionEnum.Europe}
+      ${'North America'}          | ${UsersDataRegionEnum.NorthAmerica}
+      ${'South America'}          | ${UsersDataRegionEnum.SouthAmerica}
+      ${'Latin America'}          | ${UsersDataRegionEnum.LatinAmerica}
+    `(
+      'Should update the region and last name through a clean-update $region => $expected',
```

Comment:
is there a reason why you're testing the update of region together with the name? I would test the region and the role individually, then have one test to verify that they can be all updated together too

Reply:
I think this is how the tests were copied over.